.github/ISSUE_TEMPLATE/feature_request.md

```diff
@@ -7,6 +7,8 @@ assignees: ''
 
 ---
 
+<!-- Please search for open issues that relate to the same problem before opening a new one. -->
```

Comment:
I don't think we should refer to feature requests as problems.

Reply:
Good point. Will get that fixed :)